### DISCUSSION TITLE
upperlimit does not pass optional arguments of hypotest

### DISCUSSION THREAD

**Discussion by jhvoss** [scikit-hep/pyhf#1890] [Ideas] [answered]
Dear pyhf team,

I just started using pyhf, wanted to use the upperlimit function and noticed that it does not seem to pass any of the optional arguments of the hypotest function it uses, but I would need it to pass some new par_bounds.
Would it maybe be useful in general (and not just me) to let it pass par_bounds, or also other optional arguments, to hypotest?

**Comment by kratsg** [comment, marked as answer]
@jhvoss, this is done in #1613 and should be available via the release candidate [0.7.0rc1](https://pypi.org/project/pyhf/0.7.0rc1/).

This feature should exist in v0.7.0+.
